articles/azure-arc/servers/prerequisites.md

```diff
@@ -34,6 +34,9 @@ The following versions of the Windows and Linux operating system are officially
 * Amazon Linux 2
 * Oracle Linux 7 and 8
 
+> [!NOTE] On Linux, Azure Arc-enabled Servers installs several daemon processes. We only support using systemd to manage these processes. In some environments systemd may not be installed or available, in which case Arc-enabled servers is not supported, even if the distribution is otherwise supported. These environments include **Windows Subsystem for Linux** (WSL) and most container-based systems such as Kubernetes or Docker. The azure connected machine agent can be installed on the node which runs the containers, but not inside the containers themselves.
```

Comment:
Looks good but can you please fix the capitalization - "Azure Arc-enabled servers" at the start of the note, and "Azure Connected Machine agent" in last sentence.

Reply:
> Looks good but can you please fix the capitalization - "Azure Arc-enabled servers" at the start of the note, and "Azure Connected Machine agent" in last sentence.

@ JnHs, you want the PR reviewers to make these changes, let us know (and add `#sign-off`) and we will do so. Thanks.